app.py

```diff
@@ -23,10 +25,20 @@
 def static_file(path):
     return app.send_static_file(path)
 
+AUTH_TYPE = os.environ.get("AUTH_TYPE")
```

Comment:
Please default the AUTH_TYPE to keys and then add a section to the readme on how a user would switch from keys to rbac using azd env set AUTH_TYPE rbac.  Thanks